source/user-manual/agent/agent-management/grouping-agents.rst

```diff
@@ -358,3 +358,58 @@ In the case of belonging to multiple groups, the configuration files of every gr
     :alt: Multi-group shared files
     :align: center
     :width: 70%
+
+Assigning multiple agents to a group
+------------------------------------
+
+It's also possible to assign multiple agents to a single group using the Wazuh API endpoint :api-ref:`PUT /agents/group <operation/api.controllers.agent_controller.put_multiple_agent_single_group>`. For instance, to assign agents with IDs ``001`` and ``002`` to the ``aws_agents`` group, execute the following request.
```

Comment:
Replaced `run the following command` with `execute the following request` because even though it's a command, what is actually assigning the agents is the request that the command creates.